agent/api/ecsclient/client.go

```diff
@@ -394,6 +394,12 @@ func (client *APIECSClient) SubmitTaskStateChange(change api.TaskStateChange) er
 		ExecutionStoppedAt: change.ExecutionStoppedAt,
 	}
 
+	for _, managedAgentEvent := range change.ManagedAgents {
```

Comment:
please add include this in the unit tests to verify if managed agent changes are in the task state change.